chapter07/kds/ex4.pl

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+print "Enter some text or supply an input file as an argument:\n";
+while( <> ){
+  print "$_" if( /[A-Z][a-z]/ );
```

Comment:
It would fail to match mr.T as the last letter is the capital or the A-team as there is a non-character in between the capital and non-capital.